.env.example

```diff
@@ -41,9 +59,19 @@ QFIELDCLOUD_TLS_KEY="/etc/nginx/certs/${QFIELDCLOUD_HOST}-key.pem"
 # DEFAULT: "/etc/nginx/dhparams/ssl-dhparams.pem"
 QFIELDCLOUD_TLS_DHPARAMS="/etc/nginx/dhparams/ssl-dhparams.pem"
 
+# Let's Encrypt email used for registration and recovery contact.
+# DEFAULT: test@example.com
 LETSENCRYPT_EMAIL="test@example.com"
+
+# Let's Encrypt RSA key size. See https://eff-certbot.readthedocs.io/en/stable/using.html#rsa-and-ecdsa-keys.
+# VALUES: 4096
+# DEFAULT: 4096
 LETSENCRYPT_RSA_KEY_SIZE=4096
-# Set to 1 if you're testing your setup to avoid hitting request limits
+
+# Whether the Let's Encrypt certificate is used in staging or production environment. See https://eff-certbot.readthedocs.io/en/stable/using.html#revoking-certificates.
```

Comment:
Newline before `See ...`? 
_(If this was intentional, e.g. for line-diffability, feel free to just ignore without any response 😛 )_